server/chat/backend/agent/tools/apply_fix_utils.py

```
@@ -0,0 +1,90 @@
+                    "description": row[3],
+                    "type": row[4],
+                    "file_path": row[5],
+                    "original_content": row[6],
+                    "suggested_content": row[7],
+                    "user_edited_content": row[8],
+                    "repository": row[9],
+                    "commit_message": row[10],
+                    "pr_url": row[11],
+                    "created_branch": row[12],
+                }
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)